### DISCUSSION TITLE
Css Is not Applied when i deploye the Website in NextJS

### DISCUSSION THREAD

**Discussion by Yashesh133** [vercel/community#5071] [Help]
### Summary

I have  issue, the CSS is not loading in vercel, do you mind to check for me?
my github repo: https://github.com/Yashesh133/Travel-Website-UI-UX-Design
my vercel: https://travel-website-ui-ux-design.vercel.app/

### Steps to Reproduce

I have  issue, the CSS is not loading in vercel, do you mind to check for me?
my github repo: https://github.com/Yashesh133/Travel-Website-UI-UX-Design
my vercel: https://travel-website-ui-ux-design.vercel.app/

![1121](https://github.com/vercel/community/assets/119943375/08404280-01be-43c6-b80b-5b5fa2d93c0c)

**Comment by abhishek1savaliya** [comment]
try using CDN link
